common/lib/capa/capa/tests/test_shuffle.py

```diff
@@ -58,7 +59,10 @@ def test_shuffle_custom_names(self):
         response = list(problem.responders.values())[0]
         self.assertFalse(response.has_mask())
         self.assertTrue(response.has_shuffle())
-        self.assertEqual(response.unmask_order(), ['choice_0', 'choice_aaa', 'choice_1', 'choice_ddd'])
+        if six.PY2:
+            self.assertEqual(response.unmask_order(), ['choice_0', 'choice_aaa', 'choice_1', 'choice_ddd'])
```

Comment:
Do we just want assertItemsEqual here instead?

Reply:
No, because we are testing whether the shuffle functionality works in this case. If for some reason it fails to work, assertItemsEqual would still work cause the output list will always be same regardless of whether shuffle function worked or not. The test isn't perfect. But because shuffle can technically result in same output as input, I couldn't think of a better test didn't have possibility of false negatives.

Reply:
Cool, makes sense, thanks!